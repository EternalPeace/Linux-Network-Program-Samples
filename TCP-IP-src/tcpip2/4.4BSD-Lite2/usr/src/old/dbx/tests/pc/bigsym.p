(*
 * Test of large symbol entries.
 *)

program bigsym (input, output);
type
    BigEnum = (
	A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15,
	B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B11, B12, B13, B14, B15,
	C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, C15,
	D1, D2, D3, D4, D5, D6, D7, D8, D9, D10, D11, D12, D13, D14, D15,
	E1, E2, E3, E4, E5, E6, E7, E8, E9, E10, E11, E12, E13, E14, E15,
	F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12, F13, F14, F15,
	G1, G2, G3, G4, G5, G6, G7, G8, G9, G10, G11, G12, G13, G14, G15,
	H1, H2, H3, H4, H5, H6, H7, H8, H9, H10, H11, H12, H13, H14, H15,
	I1, I2, I3, I4, I5, I6, I7, I8, I9, I10, I11, I12, I13, I14, I15,
	J1, J2, J3, J4, J5, J6, J7, J8, J9, J10, J11, J12, J13, J14, J15,
	K1, K2, K3, K4, K5, K6, K7, K8, K9, K10, K11, K12, K13, K14, K15,
	L1, L2, L3, L4, L5, L6, L7, L8, L9, L10, L11, L12, L13, L14, L15,
	M1, M2, M3, M4, M5, M6, M7, M8, M9, M10, M11, M12, M13, M14, M15,
	N1, N2, N3, N4, N5, N6, N7, N8, N9, N10, N11, N12, N13, N14, N15,
	O1, O2, O3, O4, O5, O6, O7, O8, O9, O10, O11, O12, O13, O14, O15,
	P1, P2, P3, P4, P5, P6, P7, P8, P9, P10, P11, P12, P13, P14, P15,
	Q1, Q2, Q3, Q4, Q5, Q6, Q7, Q8, Q9, Q10, Q11, Q12, Q13, Q14, Q15,
	R1, R2, R3, R4, R5, R6, R7, R8, R9, R10, R11, R12, R13, R14, R15,
	S1, S2, S3, S4, S5, S6, S7, S8, S9, S10, S11, S12, S13, S14, S15,
	T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15,
	U1, U2, U3, U4, U5, U6, U7, U8, U9, U10, U11, U12, U13, U14, U15,
	V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12, V13, V14, V15,
	W1, W2, W3, W4, W5, W6, W7, W8, W9, W10, W11, W12, W13, W14, W15,
	X1, X2, X3, X4, X5, X6, X7, X8, X9, X10, X11, X12, X13, X14, X15,
	Y1, Y2, Y3, Y4, Y5, Y6, Y7, Y8, Y9, Y10, Y11, Y12, Y13, Y14, Y15,
	Z1, Z2, Z3, Z4, Z5, Z6, Z7, Z8, Z9, Z10, Z11, Z12, Z13, Z14, Z15 
    );

    BigRec = record
	A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15,
	B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B11, B12, B13, B14, B15,
	C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, C15,
	D1, D2, D3, D4, D5, D6, D7, D8, D9, D10, D11, D12, D13, D14, D15,
	E1, E2, E3, E4, E5, E6, E7, E8, E9, E10, E11, E12, E13, E14, E15,
	F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12, F13, F14, F15,
	G1, G2, G3, G4, G5, G6, G7, G8, G9, G10, G11, G12, G13, G14, G15,
	H1, H2, H3, H4, H5, H6, H7, H8, H9, H10, H11, H12, H13, H14, H15,
	I1, I2, I3, I4, I5, I6, I7, I8, I9, I10, I11, I12, I13, I14, I15,
	J1, J2, J3, J4, J5, J6, J7, J8, J9, J10, J11, J12, J13, J14, J15,
	K1, K2, K3, K4, K5, K6, K7, K8, K9, K10, K11, K12, K13, K14, K15,
	L1, L2, L3, L4, L5, L6, L7, L8, L9, L10, L11, L12, L13, L14, L15,
	M1, M2, M3, M4, M5, M6, M7, M8, M9, M10, M11, M12, M13, M14, M15,
	N1, N2, N3, N4, N5, N6, N7, N8, N9, N10, N11, N12, N13, N14, N15,
	O1, O2, O3, O4, O5, O6, O7, O8, O9, O10, O11, O12, O13, O14, O15,
	P1, P2, P3, P4, P5, P6, P7, P8, P9, P10, P11, P12, P13, P14, P15,
	Q1, Q2, Q3, Q4, Q5, Q6, Q7, Q8, Q9, Q10, Q11, Q12, Q13, Q14, Q15,
	R1, R2, R3, R4, R5, R6, R7, R8, R9, R10, R11, R12, R13, R14, R15,
	S1, S2, S3, S4, S5, S6, S7, S8, S9, S10, S11, S12, S13, S14, S15,
	T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15,
	U1, U2, U3, U4, U5, U6, U7, U8, U9, U10, U11, U12, U13, U14, U15,
	V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12, V13, V14, V15,
	W1, W2, W3, W4, W5, W6, W7, W8, W9, W10, W11, W12, W13, W14, W15,
	X1, X2, X3, X4, X5, X6, X7, X8, X9, X10, X11, X12, X13, X14, X15,
	Y1, Y2, Y3, Y4, Y5, Y6, Y7, Y8, Y9, Y10, Y11, Y12, Y13, Y14, Y15,
	Z1, Z2, Z3, Z4, Z5, Z6, Z7, Z8, Z9, Z10, Z11, Z12, Z13, Z14, Z15
	: integer;
    end;

    BigRec2 = record
	abcdefghijklmnopqrstuvwxyz : integer;
	bcdefghijklmnopqrstuvwxyza : integer;
	cdefghijklmnopqrstuvwxyzab : integer;
	defghijklmnopqrstuvwxyzabc : integer;
	efghijklmnopqrstuvwxyzabcd : integer;
	fghijklmnopqrstuvwxyzabcde : integer;
    end;
begin
end.
