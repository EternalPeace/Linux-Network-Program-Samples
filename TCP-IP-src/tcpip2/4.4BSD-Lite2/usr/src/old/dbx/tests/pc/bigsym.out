reading symbolic information ...
type BigEnum = (A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B11, B12, B13, B14, B15, C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, C15, D1, D2, D3, D4, D5, D6, D7, D8, D9, D10, D11, D12, D13, D14, D15, E1, E2, E3, E4, E5, E6, E7, E8, E9, E10, E11, E12, E13, E14, E15, F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12, F13, F14, F15, G1, G2, G3, G4, G5, G6, G7, G8, G9, G10, G11, G12, G13, G14, G15, H1, H2, H3, H4, H5, H6, H7, H8, H9, H10, H11, H12, H13, H14, H15, I1, I2, I3, I4, I5, I6, I7, I8, I9, I10, I11, I12, I13, I14, I15, J1, J2, J3, J4, J5, J6, J7, J8, J9, J10, J11, J12, J13, J14, J15, K1, K2, K3, K4, K5, K6, K7, K8, K9, K10, K11, K12, K13, K14, K15, L1, L2, L3, L4, L5, L6, L7, L8, L9, L10, L11, L12, L13, L14, L15, M1, M2, M3, M4, M5, M6, M7, M8, M9, M10, M11, M12, M13, M14, M15, N1, N2, N3, N4, N5, N6, N7, N8, N9, N10, N11, N12, N13, N14, N15, O1, O2, O3, O4, O5, O6, O7, O8, O9, O10, O11, O12, O13, O14, O15, P1, P2, P3, P4, P5, P6, P7, P8, P9, P10, P11, P12, P13, P14, P15, Q1, Q2, Q3, Q4, Q5, Q6, Q7, Q8, Q9, Q10, Q11, Q12, Q13, Q14, Q15, R1, R2, R3, R4, R5, R6, R7, R8, R9, R10, R11, R12, R13, R14, R15, S1, S2, S3, S4, S5, S6, S7, S8, S9, S10, S11, S12, S13, S14, S15, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15, U1, U2, U3, U4, U5, U6, U7, U8, U9, U10, U11, U12, U13, U14, U15, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12, V13, V14, V15, W1, W2, W3, W4, W5, W6, W7, W8, W9, W10, W11, W12, W13, W14, W15, X1, X2, X3, X4, X5, X6, X7, X8, X9, X10, X11, X12, X13, X14, X15, Y1, Y2, Y3, Y4, Y5, Y6, Y7, Y8, Y9, Y10, Y11, Y12, Y13, Y14, Y15, Z1, Z2, Z3, Z4, Z5, Z6, Z7, Z8, Z9, Z10, Z11, Z12, Z13, Z14, Z15);
type BigRec = record
    A1 : integer;
    A2 : integer;
    A3 : integer;
    A4 : integer;
    A5 : integer;
    A6 : integer;
    A7 : integer;
    A8 : integer;
    A9 : integer;
    A10 : integer;
    A11 : integer;
    A12 : integer;
    A13 : integer;
    A14 : integer;
    A15 : integer;
    B1 : integer;
    B2 : integer;
    B3 : integer;
    B4 : integer;
    B5 : integer;
    B6 : integer;
    B7 : integer;
    B8 : integer;
    B9 : integer;
    B10 : integer;
    B11 : integer;
    B12 : integer;
    B13 : integer;
    B14 : integer;
    B15 : integer;
    C1 : integer;
    C2 : integer;
    C3 : integer;
    C4 : integer;
    C5 : integer;
    C6 : integer;
    C7 : integer;
    C8 : integer;
    C9 : integer;
    C10 : integer;
    C11 : integer;
    C12 : integer;
    C13 : integer;
    C14 : integer;
    C15 : integer;
    D1 : integer;
    D2 : integer;
    D3 : integer;
    D4 : integer;
    D5 : integer;
    D6 : integer;
    D7 : integer;
    D8 : integer;
    D9 : integer;
    D10 : integer;
    D11 : integer;
    D12 : integer;
    D13 : integer;
    D14 : integer;
    D15 : integer;
    E1 : integer;
    E2 : integer;
    E3 : integer;
    E4 : integer;
    E5 : integer;
    E6 : integer;
    E7 : integer;
    E8 : integer;
    E9 : integer;
    E10 : integer;
    E11 : integer;
    E12 : integer;
    E13 : integer;
    E14 : integer;
    E15 : integer;
    F1 : integer;
    F2 : integer;
    F3 : integer;
    F4 : integer;
    F5 : integer;
    F6 : integer;
    F7 : integer;
    F8 : integer;
    F9 : integer;
    F10 : integer;
    F11 : integer;
    F12 : integer;
    F13 : integer;
    F14 : integer;
    F15 : integer;
    G1 : integer;
    G2 : integer;
    G3 : integer;
    G4 : integer;
    G5 : integer;
    G6 : integer;
    G7 : integer;
    G8 : integer;
    G9 : integer;
    G10 : integer;
    G11 : integer;
    G12 : integer;
    G13 : integer;
    G14 : integer;
    G15 : integer;
    H1 : integer;
    H2 : integer;
    H3 : integer;
    H4 : integer;
    H5 : integer;
    H6 : integer;
    H7 : integer;
    H8 : integer;
    H9 : integer;
    H10 : integer;
    H11 : integer;
    H12 : integer;
    H13 : integer;
    H14 : integer;
    H15 : integer;
    I1 : integer;
    I2 : integer;
    I3 : integer;
    I4 : integer;
    I5 : integer;
    I6 : integer;
    I7 : integer;
    I8 : integer;
    I9 : integer;
    I10 : integer;
    I11 : integer;
    I12 : integer;
    I13 : integer;
    I14 : integer;
    I15 : integer;
    J1 : integer;
    J2 : integer;
    J3 : integer;
    J4 : integer;
    J5 : integer;
    J6 : integer;
    J7 : integer;
    J8 : integer;
    J9 : integer;
    J10 : integer;
    J11 : integer;
    J12 : integer;
    J13 : integer;
    J14 : integer;
    J15 : integer;
    K1 : integer;
    K2 : integer;
    K3 : integer;
    K4 : integer;
    K5 : integer;
    K6 : integer;
    K7 : integer;
    K8 : integer;
    K9 : integer;
    K10 : integer;
    K11 : integer;
    K12 : integer;
    K13 : integer;
    K14 : integer;
    K15 : integer;
    L1 : integer;
    L2 : integer;
    L3 : integer;
    L4 : integer;
    L5 : integer;
    L6 : integer;
    L7 : integer;
    L8 : integer;
    L9 : integer;
    L10 : integer;
    L11 : integer;
    L12 : integer;
    L13 : integer;
    L14 : integer;
    L15 : integer;
    M1 : integer;
    M2 : integer;
    M3 : integer;
    M4 : integer;
    M5 : integer;
    M6 : integer;
    M7 : integer;
    M8 : integer;
    M9 : integer;
    M10 : integer;
    M11 : integer;
    M12 : integer;
    M13 : integer;
    M14 : integer;
    M15 : integer;
    N1 : integer;
    N2 : integer;
    N3 : integer;
    N4 : integer;
    N5 : integer;
    N6 : integer;
    N7 : integer;
    N8 : integer;
    N9 : integer;
    N10 : integer;
    N11 : integer;
    N12 : integer;
    N13 : integer;
    N14 : integer;
    N15 : integer;
    O1 : integer;
    O2 : integer;
    O3 : integer;
    O4 : integer;
    O5 : integer;
    O6 : integer;
    O7 : integer;
    O8 : integer;
    O9 : integer;
    O10 : integer;
    O11 : integer;
    O12 : integer;
    O13 : integer;
    O14 : integer;
    O15 : integer;
    P1 : integer;
    P2 : integer;
    P3 : integer;
    P4 : integer;
    P5 : integer;
    P6 : integer;
    P7 : integer;
    P8 : integer;
    P9 : integer;
    P10 : integer;
    P11 : integer;
    P12 : integer;
    P13 : integer;
    P14 : integer;
    P15 : integer;
    Q1 : integer;
    Q2 : integer;
    Q3 : integer;
    Q4 : integer;
    Q5 : integer;
    Q6 : integer;
    Q7 : integer;
    Q8 : integer;
    Q9 : integer;
    Q10 : integer;
    Q11 : integer;
    Q12 : integer;
    Q13 : integer;
    Q14 : integer;
    Q15 : integer;
    R1 : integer;
    R2 : integer;
    R3 : integer;
    R4 : integer;
    R5 : integer;
    R6 : integer;
    R7 : integer;
    R8 : integer;
    R9 : integer;
    R10 : integer;
    R11 : integer;
    R12 : integer;
    R13 : integer;
    R14 : integer;
    R15 : integer;
    S1 : integer;
    S2 : integer;
    S3 : integer;
    S4 : integer;
    S5 : integer;
    S6 : integer;
    S7 : integer;
    S8 : integer;
    S9 : integer;
    S10 : integer;
    S11 : integer;
    S12 : integer;
    S13 : integer;
    S14 : integer;
    S15 : integer;
    T1 : integer;
    T2 : integer;
    T3 : integer;
    T4 : integer;
    T5 : integer;
    T6 : integer;
    T7 : integer;
    T8 : integer;
    T9 : integer;
    T10 : integer;
    T11 : integer;
    T12 : integer;
    T13 : integer;
    T14 : integer;
    T15 : integer;
    U1 : integer;
    U2 : integer;
    U3 : integer;
    U4 : integer;
    U5 : integer;
    U6 : integer;
    U7 : integer;
    U8 : integer;
    U9 : integer;
    U10 : integer;
    U11 : integer;
    U12 : integer;
    U13 : integer;
    U14 : integer;
    U15 : integer;
    V1 : integer;
    V2 : integer;
    V3 : integer;
    V4 : integer;
    V5 : integer;
    V6 : integer;
    V7 : integer;
    V8 : integer;
    V9 : integer;
    V10 : integer;
    V11 : integer;
    V12 : integer;
    V13 : integer;
    V14 : integer;
    V15 : integer;
    W1 : integer;
    W2 : integer;
    W3 : integer;
    W4 : integer;
    W5 : integer;
    W6 : integer;
    W7 : integer;
    W8 : integer;
    W9 : integer;
    W10 : integer;
    W11 : integer;
    W12 : integer;
    W13 : integer;
    W14 : integer;
    W15 : integer;
    X1 : integer;
    X2 : integer;
    X3 : integer;
    X4 : integer;
    X5 : integer;
    X6 : integer;
    X7 : integer;
    X8 : integer;
    X9 : integer;
    X10 : integer;
    X11 : integer;
    X12 : integer;
    X13 : integer;
    X14 : integer;
    X15 : integer;
    Y1 : integer;
    Y2 : integer;
    Y3 : integer;
    Y4 : integer;
    Y5 : integer;
    Y6 : integer;
    Y7 : integer;
    Y8 : integer;
    Y9 : integer;
    Y10 : integer;
    Y11 : integer;
    Y12 : integer;
    Y13 : integer;
    Y14 : integer;
    Y15 : integer;
    Z1 : integer;
    Z2 : integer;
    Z3 : integer;
    Z4 : integer;
    Z5 : integer;
    Z6 : integer;
    Z7 : integer;
    Z8 : integer;
    Z9 : integer;
    Z10 : integer;
    Z11 : integer;
    Z12 : integer;
    Z13 : integer;
    Z14 : integer;
    Z15 : integer;
end;
type BigRec2 = record
    abcdefghijklmnopqrstuvwxyz : integer;
    bcdefghijklmnopqrstuvwxyza : integer;
    cdefghijklmnopqrstuvwxyzab : integer;
    defghijklmnopqrstuvwxyzabc : integer;
    efghijklmnopqrstuvwxyzabcd : integer;
    fghijklmnopqrstuvwxyzabcde : integer;
end;

